pkg/ebpf/c/tracee.bpf.c

```diff
@@ -579,6 +579,7 @@ BPF_HASH(sys_32_to_64_map, u32, u32);                   // map 32bit to 64bit sy
 BPF_HASH(params_types_map, u32, u64);                   // encoded parameters types for event
 BPF_HASH(process_tree_map, u32, u32);                   // filter events by the ancestry of the traced process
 BPF_HASH(process_context_map, u32, process_context_t);  // holds the process_context data for every tid
+BPF_HASH(symbols_map, u32, u64)                         // holds the addresses of kernel symbols
```

Comment:
let's call it ksymbols_map or kernel_symbols_map to signify the origin of the symbols (we might have userspace symbols in the future as well).
Also, maybe change the comment to:
"holds the addresses of **some** kernel symbols"